blog/2024-09-17-announcing-angular-sdk.md

```diff
@@ -0,0 +1,126 @@
+---
+slug: 'announcing-angular-sdk'
+title: 'Announcing the OpenFeature Angular SDK!'
+date: 2024-09-17
+authors: [lukasreining, toddbaert]
+image: /img/blog/2024-09-17-announcing-angular-sdk/angular.png
+description: "Announcing the first official release of the OpenFeature Angular SDK!"
+tags: [angular, ng, sdk, ts, js, javascript, web]
+draft: false
+---
+
+We are excited to unveil the first official release of the [@openfeature/angular-sdk](/docs/reference/technologies/client/web/angular)! 🚀 
+This SDK extends OpenFeature capabilities to Angular applications, with a focus on Angular's unique patterns and practices.
+In this post, we’ll walk you through some of the standout features and how they integrate seamlessly with Angular development.
```

Comment:
Should we make it clear that this is an early-access SDK? This reads like it's a stable release to me.

Reply:
I considered something like that, but I think that's more appropriate for docs/READMEs than blog posts which are about HYPE.

I think this has a similar tone and phrasing as the react SDK post.

Reply:
Makes sense to me as @toddbaert says.